server/src/com/thoughtworks/go/config/CachedFileGoConfig.java

```diff
@@ -230,9 +235,11 @@ private synchronized void notifyListeners(CruiseConfig newCruiseConfig) {
         LOGGER.info("About to notify config listeners");
         for (ConfigChangedListener listener : listeners) {
             try {
+                long startTime = System.currentTimeMillis();
                 listener.onConfigChange(newCruiseConfig);
+                LOGGER.debug("Notifying {} took (in ms): {}", listener.getClass(), (System.currentTimeMillis() - startTime));
             } catch (Exception e) {
-                LOGGER.error("failed to fire config changed event for listener: " + listener, e);
+                LOGGER.error("Failed to fire config changed event for listener: " + listener, e);
```

Comment:
Since this is logging an exception, not using the `{}` syntax. Also, don't think this happens often enough for `String.format` to make any real difference.